### PR TITLE
Add retrieving of stream properties to RESTful API.

### PR DESCRIPTION
**Pull request by awholegunch**
Fixes [CDAP-1914](https://issues.cask.co/browse/CDAP-1914).

Staged at: [Getting and Setting Stream Properties](http://docs-staging.cask.co/staging/cdap/3.2.0-SNAPSHOT-feature-3.2_1914_Stream_API_docs/en/reference-manual/http-restful-api/stream.html#getting-and-setting-stream-properties)

### CODE REVIEW

File: cdap-docs/reference-manual/source/http-restful-api/stream.rst [hunk 1]
```diff
@@ -340,19 +340,69 @@ A stream can be deleted with an HTTP DELETE method to the URL::
...
-The notification threshold defines the increment of data that a stream has to receive before
+
+The **notification threshold** defines the increment of data that a stream has to receive before
 publishing a notification.
 
+.. rubric:: Getting Stream Properties
+
+Stream properties can be retrieved with an HTTP PUT method to the URL::
+
+  GET <base-url>/namespaces/<namespace>/streams/<stream-id>/
```
Comment (gokulavasan) [inline cdap-docs/reference-manual/source/http-restful-api/stream.rst]: nit: remove '/' at end? feel free to ignore.

File: cdap-docs/reference-manual/source/http-restful-api/stream.rst [hunk 2]
```diff
@@ -340,19 +340,69 @@ A stream can be deleted with an HTTP DELETE method to the URL::
...
+           "format": {
+             "name": "text",
+             "schema": {
+               "type": "record",
+               "name": "stringBody",
+               "fields": [
+                 { "name": "body", "type": "string" }
+               ]
+             },
+             "settings": {}
```
Comment (gokulavasan) [inline cdap-docs/reference-manual/source/http-restful-api/stream.rst]: is it ok if we don't talk about this ``settings`` property?

Comment (gokulavasan) [inline cdap-docs/reference-manual/source/http-restful-api/stream.rst]: oh I see an example below.

**Comment by gokulavasan** [comment]
Couple of comments. Can be ignored. LGTM :+1:

File: cdap-docs/reference-manual/source/http-restful-api/stream.rst (hunk 1 above)
Comment (awholegunch) [inline cdap-docs/reference-manual/source/http-restful-api/stream.rst]: fixed.

**Pull request merged**